runtime/development/src/constants.rs

```diff
@@ -0,0 +1,15 @@
+//! A set of constant values used in altair runtime
```

Comment:
Since the value conversion should be the same among runtimes, do you think we can just create an alias  from CFG?